src/pages/Introduce/Introduce.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { Link, useNavigate } from "react-router-dom";
+import "../Introduce/Introduce.scss";
+
+function Introduce() {
+  const navigate = useNavigate();
+
+  const handleButton = () => {
+    console.log("button 태그 동작");
+    navigate("/product");
+  };
+
+  const handleInput = () => {
+    console.log("input 태그 동작");
+  };
+
+  return (
+    <div className="introduce" style={{ boxSizing: "border-box" }}>
```

Comment:
```suggestion
    <div className="introduce">
```
style이 필요하다면 scss 파일에 적용해주세요!